.github/workflows/test-suite.yml

```diff
@@ -84,4 +84,42 @@ jobs:
     if: ${{ !cancelled() }}
     secrets: inherit
     uses: ./.github/workflows/cleanup-test-env.yml
-  
\ No newline at end of file
+  
+  slack-notification:
+    needs: [mig-tests, acc-tests, clean-after]
+    if: ${{ (github.event_name == 'schedule' || github.event_name == 'workflow_dispatch') && (needs.mig-tests.result == 'failure' || needs.acc-tests.result == 'failure')}}
```

Comment:
wouldn't schedule be enough?

Reply:
I think having the notification also if the action is manually triggered can be useful.